components/NavigationDrawer/src/MDCBottomDrawerTransitionController.h

```diff
@@ -28,4 +29,9 @@
  */
 @property(nonatomic, weak, nullable) UIScrollView *trackingScrollView;
 
+/**
+ The current state of the bottom drawer.
+ */
+@property(nonatomic, readonly) MDCBottomDrawerState state;
```

Comment:
Nit/naming: Since this is the state of another object (the BottomDrawer), please rename it to something like, `drawerState`. Having it named `state` feels like it's the transition controller's state.